uuid/uuid.py

```diff
@@ -0,0 +1,34 @@
+'''Implements basic interface of UUID and uuid4()
+'''
+import os
+import ubinascii
+
+
+class UUID:
+    def __init__(self, bytes=None):
+        if bytes:
+            if len(bytes) != 16:
+                raise ValueError('bytes arg must be 16 bytes long')
+            self._bytes = bytes
+
+        else:
+            raise ValueError("No valid argument passed")
+
+    @property
+    def hex(self):
+        return ubinascii.hexlify(self._bytes).decode()
+            
```

Comment:
Github doesn't show that explicitly (well, you can try to select it with mouse drag), but there's trailing whitespace on this (otherwise empty) line, can you take care of it?